src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2118,7 +2134,9 @@ export default class Aircraft {
                 this.target.heading = vradial(vector_to_fix) - Math.PI;
                 this.target.turn = null;
             }
-        } else if (this.fms.currentWaypoint().navmode === WAYPOINT_NAV_MODE.HOLD) {
+        }
+
+        else if (this.fms.currentWaypoint().navmode === WAYPOINT_NAV_MODE.HOLD) {
```

Comment:
make sure an else if is on the same line as the closing brace for the previous conditional.